Sources/Models/LocationMessageSnapshotOptions.swift

```diff
@@ -27,24 +27,38 @@ import MapKit
 /// An object grouping the settings used by the `MKMapSnapshotter` through the `LocationMessageDisplayDelegate`.
 public struct LocationMessageSnapshotOptions {
 
+    /// Initialize LocationMessageSnapshotOptions with given parameters
+    ///
+    /// - Parameters:
+    ///   - showsBuildings: A Boolean value indicating whether the snapshot image should display buildings.
+    ///   - showsPointsOfInterest: A Boolean value indicating whether the snapshot image should display points of interest.
+    ///   - span: The span of the snapshot.
+    ///   - scale: The scale of the snapshot.
+    public init(showsBuildings: Bool = false, showsPointsOfInterest: Bool = false, span: MKCoordinateSpan = MKCoordinateSpan(latitudeDelta: 0, longitudeDelta: 0) , scale: CGFloat = UIScreen.main.scale) {
+        self.showsBuildings = showsBuildings
+        self.showsPointsOfInterest = showsPointsOfInterest
+        self.span = span
+        self.scale = scale
+    }
+    
     /// A Boolean value indicating whether the snapshot image should display buildings.
     ///
     /// The default value of this property is `false`.
-    var showsBuildings = false
-
+    public var showsBuildings = false
```

Comment:
Please remove the initial value, because we have the default value in `initializer` method.